Tests/iaas/security-groups/default-security-group-rules.py

```diff
@@ -82,34 +85,125 @@ def test_rules(cloud_name: str):
         f"Expected only ingress rules for default security groups, "
         f"that allow ingress traffic from the same group. "
         f"But there are more - in total {ingress_rules} ingress rules. "
-        f"There should be only {ingress_from_same_sg} ingress rules.")
-    assert egress_rules > 0, (
-        f"Expected to have more than {egress_rules} egress rules present.")
-    var_list = [egress_ipv4_default_sg, egress_ipv4_custom_sg,
-                egress_ipv6_default_sg, egress_ipv6_custom_sg]
+        f"There should be only {ingress_from_same_sg} ingress rules."
+    )
+    assert (
+        egress_rules > 0
+    ), f"Expected to have more than {egress_rules} egress rules present."
+    var_list = [
+        egress_ipv4_default_sg,
+        egress_ipv4_custom_sg,
+        egress_ipv6_default_sg,
+        egress_ipv6_custom_sg,
+    ]
     assert all([var > 0 for var in var_list]), (
         "Not all expected egress rules are present. "
         "Expected rules for egress for IPv4 and IPv6 "
-        "both for default and custom security groups.")
+        "both for default and custom security groups."
+    )
 
-    result_dict = {
-        "Ingress Rules": ingress_rules,
-        "Egress Rules": egress_rules
-    }
+    result_dict = {"Ingress Rules": ingress_rules, "Egress Rules": egress_rules}
+    return result_dict
+
+
+def create_security_group(
+    conn, sg_name: str = "default-test-sg", description: str = "default-test-sg"
+):
```

Comment:
The only added value of this function is the default values for the arguments?

Reply:
i turned the default values into globals, now or would you recommend to get rid of this function completely?